video/out/vo_dmabuf_wayland.c

```diff
@@ -244,6 +236,13 @@ static void draw_frame(struct vo *vo, struct vo_frame *frame)
     if (!entry)
         return;
 
+    // ensure the pool is reset after hwdec seek,
+    // to avoid stutter artifact
+    if (p->want_reset)
+        wlbuf_pool_clean(p->wlbuf_pool,false);
+    if (p->want_resize)
+        resize(vo);
+
     MP_VERBOSE(entry->vo, "Schedule buffer pool entry : %lu\n",entry->key );
```

Comment:
Woops missed this. Could you make this `MP_TRACE` as well since it's per frame?

Reply:
thx, fixed.